matsim/pom.xml

```diff
@@ -318,6 +262,13 @@
 			<groupId>com.google.inject</groupId>
 			<artifactId>guice</artifactId>
 			<version>${guice.version}</version>
+			<!-- Use newer guava version -->
+			<exclusions>
```

Comment:
do we need this exclusion given we have now have the guava version specified in `dependencyManagement`?